docs/src/operate/c8y/connect-mqtt-service.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 🚧 Connecting to Cumulocity MQTT Service
+tags: [Operate, Cloud, Connection, Cumulocity]
+description: Connecting %%te%% to Cumulocity
+sidebar_position: 1
+draft: true
+---
+
+import UserContext from '@site/src/components/UserContext';
+import UserContextForm from '@site/src/components/UserContextForm';
+import BrowserWindow from '@site/src/components/BrowserWindow';
+
+:::tip
+#### User Context {#user-context}
+
+You can customize the documentation and commands shown on this page by providing relevant settings which will be reflected in the instructions. It makes it even easier to explore and use %%te%%.
+
+<UserContextForm settings="DEVICE_ID,C8Y_URL,C8Y_TENANT_ID,C8Y_USER,C8Y_PASSWORD" />
+
+The user context will be persisted in your web browser's local storage.
+:::
+
+## MQTT Service
+
+Cumulocity supports a generic [MQTT service endpoint](https://cumulocity.com/docs/device-integration/mqtt-service)
+that allows users to send free-form messages (any arbitrary topic and payload) to it.
+It does not replace the existing core MQTT endpoint and hence needs to be connected to, separately.
+The following sections cover steps required to establish a connection to the Cumulocity MQTT service endpoint.
+
+:::caution
+Cumulocity MQTT service is still a work-in-progress and the external interfaces are bound to change in future.
+The %%te%% interfaces might also change accordingly.
+:::
+
+### Configure the device
+
+Most of the configurations used to connect to the MQTT service endpoint of Cumulocity are same as
+the ones used to connect to the core MQTT endpoint.
+
+1. Configure Cumulocity tenant URL:
+
+   <UserContext>
+   
+   ```sh
+   sudo tedge config set c8y.url $C8Y_URL
+   ```
+   
+   </UserContext>
+
+   :::note
+   Though you're setting the `c8y.url` config, the `c8y.mqtt_service.url` config is used under-the-hood for the connection,
+   as this config is derived from the `c8y.mqtt` config, which is further derived from `c8y.url`, by default.
+   For example, when `c8y.url` is `example.cumulocity.com`, `c8y.mqtt` would be `example.cumulocity.com:8883`
+   and `c8y.mqtt_service.url` would be `example.cumulocity.com:9883`.
+   If the MQTT service url is different from the one that would be derived from `c8y.url` or `c8y.mqtt`,
+   set`c8y.mqtt_service.url` explicitly.
+   :::
+
+1. If you're using device certificates to connect to Cumulocity, set the tenant id (skip this when using username/password):
+
+   <UserContext>
+   
+   ```sh
+   sudo tedge config set c8y.tenant_id $C8Y_TENANT_ID
+   ```
+   
+   </UserContext>
+
+1. Enable connection to MQTT service endpoint:
+
+   ```sh
+   sudo tedge config set c8y.mqtt_service.enabled true
+   ```
+
+1. Provide a topic to subscribe to (only a single topic is allowed at the moment):
```

Comment:
Is there is a visible error if the bridge tries to subscribe to multiple topics? As I noticed that the system tests are using multiple topics (at least the mosquitto bridge test, but I also changed the in-built bridge tests to use multiple topics and it connects fine now...though given the lack of validation it is difficult to say if it actually will receive messages on that topic).

E.g. the mosquitto bridge uses the following config.

```
tedge config set c8y.mqtt_service.topics 'sub/topic,demo/topic'
```